axelnguyen0701/book-exchange
# Bidding front end axel

## Changes
- User now can create, view, accept, and confirm bid
## Testing steps

1. On 1st account, create an auction listing
2. On 2nd account, create a new bid
3. (Optional) On a 3rd account, create another new bid
4. On 1st account, accept the highest bid
5. If there is an optional 3rd bid then on that 3rd account, confirm the chosen bid. Otherwise, use 2nd account to confirm bid.
6. Go to my books to check if the book is transfered.
